nflow-engine/src/test/java/io/nflow/engine/internal/executor/WorkflowStateProcessorTest.java

```diff
@@ -100,10 +99,9 @@
 public class WorkflowStateProcessorTest extends BaseNflowTest {
 
   /**
-   * TODO junit5 doesn't have global timeouts yet
-   * https://stackoverflow.com/questions/47041313/junit-5-global-timeout
-   * @Rule
-   * public Timeout timeoutPerMethod = Timeout.seconds(5);
+   * TODO junit5 doesn't have global timeouts yet https://stackoverflow.com/questions/47041313/junit-5-global-timeout
```

Comment:
Is this comment still needed? I think I added timeouts back to many tests already